meerk40t/gui/scenewidgets/attractionwidget.py

```diff
@@ -259,7 +272,9 @@ def process_draw(self, gc):
                     if pts[2] in (TYPE_POINT, TYPE_BOUND):
                         self.draw_caret(gc, pts[0], pts[1], closeup)
                     elif pts[2] == TYPE_MIDDLE:
-                        self.draw_midpoint(gc, pts[0], pts[1], closeup)
+                        self.draw_midpoint(gc, pts[0], pts[1], closeup, False)
```

Comment:
def draw_midpoint(self, gc, x, y, closeup):

What's the False or True there for? Is that *also* closeup?

This will crash.

Reply:
That's an artefact from a test - and yes it will crash - fixed